ovh/data_dbaas_logs.go

```diff
@@ -0,0 +1,139 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"net/url"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceDbaasLogs() *schema.Resource {
+	return &schema.Resource{
+		Read: func(d *schema.ResourceData, meta interface{}) error {
+			return dataSourceDbaasLogsRead(d, meta)
+		},
+		Schema: map[string]*schema.Schema{
+			"service_name": {
+				Type:        schema.TypeString,
+				Description: "The service name",
+				Required:    true,
+			},
+			// Computed
+			"cluster_type": {
+				Type:        schema.TypeString,
+				Description: "Cluster type",
+				Computed:    true,
+			},
+			"dedicated_input_pem": {
```

Comment:
This attribute (and other with sensitive informations) should be set to sensitive :-)

Why?

because now we can see sensitive information clearly:

```
$ terraform apply
ovh_dbaas_logs_cluster.ldp: Refreshing state... [id=xxxxxx-xxxx-xxxx-xxxx-xxxxxxxx]

Changes to Outputs:
  + dedicated_input_pem = <<-EOT
        -----BEGIN CERTIFICATE-----
        MIIF4zCCA8ugAwIBAgIJAK/HFqmambkjdsgfqdgsfhgdsfqsfgdsq<censored>
        -----END CERTIFICATE-----
    EOT

You can apply this plan to save these new output values to the Terraform state, without changing any real infrastructure.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.
  ```

Reply:
A public certificate is not a sensitive information